checker-qual/src/main/java/org/checkerframework/checker/optional/qual/MaybePresent.java

```diff
@@ -16,6 +18,7 @@
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
+@RelevantJavaTypes(Optional.class)
```

Comment:
`@RelevantJavaTypes` is not a meta-annotation; it goes on the `BaseTypeChecker` subclass. (https://checkerframework.org/api/org/checkerframework/framework/qual/RelevantJavaTypes.html)